### ISSUE TITLE
对于ctc 中merge_repeated参数True or False的疑问

### ISSUE THREAD

**Issue by SongOfIceAndFire** [jimmyleaf/ocr_tensorflow_cnn#12]
作者你好，你的代码ctc使用decoded, log_prob = tf.nn.ctc_beam_search_decoder(logits, seq_len, merge_repeated=False) 。为什么merge_repeated使用False呢？不是应该将重复连续的字符合并吗？
期待您的回复。
